resources/lemmas.md

```diff
@@ -342,10 +321,8 @@ These lemmas abstract some properties about `#sizeWordStack`:
 
 ```k
     rule #sizeWordStack ( _ , _ ) >=Int 0 => true [smt-lemma]
-    rule #sizeWordStack ( WS , N:Int )
-      => #sizeWordStack ( WS , 0 ) +Int N
-      requires N =/=K 0
-      [lemma]
+
+    rule WS ++ .WordStack => WS
```

Comment:
i'm not sure if this is safe to add.
note that #take and #drop actually does the opposite direction rewriting, from BUF => BUF ++ .WordStack.
do you mean here `notBool #isBuf(WS)`?

Reply:
Update the evm-semantics to be compatible with the rule.